llvm/test/CodeGen/RISCV/rvv/fixed-vectors-int-buildvec.ll

```diff
@@ -1267,43 +1267,53 @@ define <16 x i8> @buildvec_v16i8_loads_contigous(ptr %p) {
 ;
 ; RVA22U64-LABEL: buildvec_v16i8_loads_contigous:
 ; RVA22U64:       # %bb.0:
-; RVA22U64-NEXT:    addi a6, a0, 8
-; RVA22U64-NEXT:    lbu t6, 1(a0)
+; RVA22U64-NEXT:    lbu a1, 1(a0)
```

Comment:
Should we test RVA22U32?

Reply:
Not sure it adds any useful coverage, but happy to add it if desired.

Reply:
I asked specifically because it allows ELEN=64 with XLen=32.

Reply:
We don't appear to have support in tree for rva22u32 (as an mattr alias).  I went digging through profile docs, and I can't find any mention of the rva22u32 profile variants; rva22 appears to only define 64 bit versions.

Do you have a particular arch string for a configuration which exercises the case you're concerned about here?

Reply:
From your other comment, it seems like plain rv32 + v + zba + zbb might be enough here?  Can you confirm?

Reply:
I hadn't realized that that the RVA* profiles are 64-bit only. rv32 + v + zba + zbb is enough.